Public/Src/Cache/ContentStore/Library/FileSystem/PassThroughFileSystem.cs

```diff
@@ -279,7 +279,11 @@ public void DeleteFile(AbsolutePath path)
                 // 0x20 is shared violation.
                 if (e.InnerException is NativeWin32Exception win32 && win32.ErrorCode == 0x20)
                 {
-                    throw new UnauthorizedAccessException(e.Message, e);
+                    var extraMessage = FileUtilities.TryFindOpenHandlesToFile(path.ToString(), out var info, printCurrentFilePath: false)

```

Comment:
>TryFindOpenHandlesToFile [](start = 53, length = 24)

This is now the third instance I see this pattern. Do we need a single `FileUtilities.MakeUnauthoriazedAccessExceptionUsable(.....)`